pynumdiff/utils/evaluate.py

```diff
@@ -89,125 +62,59 @@ def plot(x, dt, x_hat, dxdt_hat, x_truth, dxdt_truth, xlim=None, ax_x=None, ax_d
         print('RMS error in velocity: ', rms_dxdt)
 
 
-def __rms_error__(a, e):
-    """
-    Calculate rms error
-
-    :param a: the first array
-    :param e: the second array
-    :return: a float number representing the rms error
-    """
-    if _np.max(_np.abs(a-e)) > 1e16:
-        return 1e16
-    s_error = _np.ravel((a - e))**2
-    ms_error = _np.mean(s_error)
-    rms_error = _np.sqrt(ms_error)
-    return rms_error
-
-
-def metrics(x, dt, x_hat, dxdt_hat, x_truth=None, dxdt_truth=None, padding=None):
-    """
-    Evaluate x_hat based on various metrics, depending on whether dxdt_truth and x_truth are known or not.
-
-    :param x: time series that was differentiated
-    :type x: np.array
-
-    :param dt: time step in seconds
-    :type dt: float
-
-    :param x_hat: estimated (smoothed) x
-    :type x_hat: np.array
+def metrics(x, dt, x_hat, dxdt_hat, x_truth=None, dxdt_truth=None, padding=0):
```

Comment:
padding was default removing 2.5% on either end of the data. I think default behavior should be to calculate the metric on the full data, so I've changed the default to 0. If called with `None` or `'auto'`, 2.5% automatic slicing is available.